packages/react-server/src/render/error/fallback-error-markup.ts

```diff
@@ -0,0 +1,68 @@
+export const fallbackErrorMarkup = `
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/@shopify/polaris@4.24.0/styles.min.css"
+    />
+    <title>Server Error</title>
+  </head>
+  <body>
+    <div
+      style="
+        --top-bar-background: #00848e;
+        --top-bar-background-lighter: #1d9ba4;
+        --top-bar-color: #f9fafb;
+        --p-frame-offset: 0px;
+      "
+    >
+      <div class="Polaris-Page Polaris-Page--fullWidth">
+        <div class="Polaris-Page__Content">
+          <div
+            style="
+              --top-bar-background: #00848e;
+              --top-bar-background-lighter: #1d9ba4;
+              --top-bar-color: #f9fafb;
+              --p-frame-offset: 0px;
+            "
+          >
+            <div class="Polaris-EmptyState Polaris-EmptyState--withinPage">
+              <div class="Polaris-EmptyState__Section">
+                <div class="Polaris-EmptyState__DetailsContainer">
+                  <div class="Polaris-EmptyState__Details">
+                    <div class="Polaris-TextContainer">
+                      <p
+                        class="Polaris-DisplayText Polaris-DisplayText--sizeMedium"
+                      >
+                        Server error
+                      </p>
+                      <div class="Polaris-EmptyState__Content">
+                        <p>
+                          We received an error from the server. Please check
+                          again later.
+                        </p>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+                <div class="Polaris-EmptyState__ImageContainer">
+                  <img
+                    src="https://cdn.shopify.com/s/files/1/0757/9955/files/empty-state.svg"
```

Comment:
Note, this is the not right empty state image I plan to use for the fallback error page. 

**wrong one:**
![image](https://user-images.githubusercontent.com/6130700/83721607-6a815a00-a609-11ea-9930-ca159e848242.png)
 

**right one:**
![image](https://user-images.githubusercontent.com/6130700/83721668-88e75580-a609-11ea-9212-8a766681c95d.png)


### Why aren't you using the right one?

I put the expected svg in a [google cloud storage](https://console.cloud.google.com/storage/browser/cdn.shopifycloud.com/web-foundation/images). Getting this image publicly accessible is WIP.

Reply:
PR has been updated